### PR TITLE
Add translation files (pl-PL, Polish)

### PR DESCRIPTION
**Pull request by Kaligula0**
Polish (pl-PL) translation files – ready for use. Yay.

### CODE REVIEW

File: app/src/main/res/values-pl/arrays.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		<item>Lista Życzeń</item>
+		<item>Chcę Zagrać</item>
+		<item>Zamówione</item>
+		<item>Zagrane</item>
+		<item>Ocenione</item>
+		<item>Skomentowane</item>
+		<item>Mam Części</item>
+		<item>Szukam Części</item>
+	</string-array>
+	<string-array name="pref_sync_status_values">
```
Comment (ccomeaux) [inline app/src/main/res/values-pl/arrays.xml:19]: Remove this array from the file. The values will be supplied by the default arrays in `values`.

File: app/src/main/res/values-pl/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+		<item>wishlist</item>
+		<item>wanttoplay</item>
+		<item>preordered</item>
+		<item>played</item>
+		<item>rated</item>
+		<item>comment</item>
+		<item>hasparts</item>
+		<item>wantparts</item>
+	</string-array>
+	<string-array name="pref_sync_status_default">
```
Comment (ccomeaux) [inline app/src/main/res/values-pl/arrays.xml:34]: Remove this array from the file.

File: app/src/main/res/values-pl/arrays.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		<item>W Posiadaniu</item>
+		<item>Kiedyś W Posiadaniu</item>
+		<item>Na Sprzedaż</item>
+		<item>Chcę Odkupić</item>
+		<item>Chcę Kupić</item>
+		<item>Lista Życzeń</item>
+		<item>Chcę Zagrać</item>
+		<item>Zamówione</item>
+	</string-array>
+	<string-array name="collection_status_filter_values">
```
Comment (ccomeaux) [inline app/src/main/res/values-pl/arrays.xml:47]: Remove this array from the file.

File: app/src/main/res/values-pl/arrays.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+		<item>want_to_play</item>
+		<item>preordered</item>
+	</string-array>
+	<string-array name="expansion_status_filter">
+		<item>Wszystkie</item>
+		<item>Gry podstawowe</item>
+		<item>Rozszerzenia</item>
+		<item>Akcesoria</item>
+	</string-array>
+	<string-array name="expansion_status_filter_values">
```
Comment (ccomeaux) [inline app/src/main/res/values-pl/arrays.xml:63]: Remove this array from the file.

File: app/src/main/res/values-pl/strings_prefs.xml [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+	<string name="pref_about_authors">Autorzy</string>
+	<string name="pref_about_licenses">Licencje otwarte (open source)</string>
+	<string name="pref_about_contact_us">Skontaktuj się z nami</string>
+	<string name="pref_about_contact_us_summary">bgg4android@gmail.com</string>
+	<string name="pref_feedback_title">Opinia</string>
+	<string name="pref_about_privacy_policy">Polityka Prywatności</string>
+	<string name="pref_about_trello_board">Tablica Trello</string>
+	<string name="pref_about_bgg_guild">Gildia BGG for Android</string>
+
+	<string name="intent_action_account">com.boardgamegeek.intent.action.ACCOUNT</string>
```
Comment (ccomeaux) [inline app/src/main/res/values-pl/strings_prefs.xml:108]: Remove this and the next 2 strings.

Comment (ccomeaux) [reply app/src/main/res/values-pl/strings_prefs.xml:108]: Actually, remove everything in the file from here down.

**Comment by Vinne2** [comment]
1) There are some plurals that I had to use `<item quantity="few">` in. In Polish few = "numbers ending 2, 3, or 4 but not 12, 13, or 14" – as stated on https://developer.android.com/guide/topics/resources/string-resource#Plurals (and I agree)
2) If (in plurals) there's only one item left ("other") – should this be left still as plural or become a string? I believe that plural, but thought I'd better ask.
3) There are 2 lines in strings.xml (559 and 563), where I see this should be a plural in Polish (quantities "one" and "other"). Should I: a) add there a `tools:ignore="PluralsCandidate"` attribute or b) this attribute should be added also in "/values" directory or c) should we make this plural in both places *now* d) should we do *nothing* now?
4) Line 73 (string "or"). It's used in `CollectionStatusFilterDialog.kt` and `PlayStatsFragment.kt`. The case is, when strictly translated it sounds perfect in the 1st use and lets say "O.K." in latter. Well, it can work as it is. But will definitely sound better in Polish if this string is separated into two strings: `"or"→pl:"lub"` (for 1st use) and `"nor"→pl:"ani"` (for 2nd use). I can make it in this pull request (add new string here, new string in `/values` and edit `PlayStatsFragment.kt`).

**Comment by ccomeaux** [comment]
Looks like I merged too early. @Kaligula0 - for plurals, we need to provide `one`, `few, and `many` for all entries. `other` appears to be optional.

**Pull request merged**